external/resources/resources.go

```diff
@@ -200,6 +201,13 @@ func PublicInit() {
 		}
 	}
 
+	for _, v := range e.EPCC_CLI_DISABLE_RESOURCES {
```

Comment:
Maybe move this out actually, since it breaks so much, and do this in the command level.